crm_deduplicate_by_website/tests/test_crm_deduplicate_by_website.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.tools.safe_eval import safe_eval
+
+
+class TestDeduplicateByWebsite(common.TransactionCase):
+    def setUp(self):
+        super(TestDeduplicateByWebsite, self).setUp()
+        self.partner_1 = self.env['res.partner'].create({
```

Comment:
pls use `with_context(tracking_disable=True)`.
Also, if you don't need partners to be recreated for each tests it would be nice to move to SavepointCase as base class.
Each of these changes will speed up local testing and travis builds aaaaa lot :wink: